content/en/security/threats/workload_security_rules/custom_rules.md

```diff
@@ -42,78 +46,117 @@ For more information, see [CSM Threats Detection Rules][7].
-  - For steps on this method, see [Create a custom agent rule](#create-a-custom-agent-rule) and [Create a custom detection rule](#create-a-custom-detection-rule).
+  - For steps on this method, see [Create a custom agent rule][2] and create a custom detection rule.
 
 ## Create the custom Agent and detection rules together
 
-The **Assisted rule creator** option helps you create the Agent and dependent detection rules together, and ensures that the Agent rule is referenced in the detection rules. Using this tool is faster than the advanced method of creating the Agent and detection rules separately.
+CSM custom Agent rules are grouped into policies. Policies group Agent rules to help you apply multiple rules more efficiently.
 
-As you define the rules using this tool, the threat expressions generated for these rules are displayed in the tool.
+You can create and deploy different custom policies containing rules you want to apply to different sets of hosts in your infrastructure.
 
-To use the simple rule creator:
+For example, you can do the following with policies:
 
-1. In [Agent Configuration][4] or [Threat Detection Rules][3], select **New Rule**, and then select **Assisted rule creator**.
-2. Define the detection. To monitor your resource effectively, you have the following detection type options:
-   - To detect nonstandard and suspicious changes to files, select **File integrity monitoring (FIM)**.
-   - To track and analyze system software processes for malicious behavior or policy violations, select **Process activity monitoring**.
-   - Enter the file/process names or paths to monitor. 
-3. Specify more conditions. Enter any arguments to add to the threat rule expression. For example, the argument `foo` is added as `process.argv in ["foo"]`.
-4. Set severity and notification lists. 
-   - Select the severity for the signal generated when this threat is detected. 
-   - Select notification lists to notify when a signal is generated.
-5. Add the rule name and description.
-   
-   Here's an example of a new FIM rule, including the expressions generated for each rule.
+- Deploy multiple rules to a specific subset of an infrastructure using tags.
+- Assign a priority to your policies. Priorities can account for policies with partially overlapping tags.
 
-    {{< img src="/security/csm/csm_threats_simple_rule_creator2.png" alt="Assisted rule creator example" style="width:100%;" >}}
+### Default policy
 
-6. Select **Create _N_ Rules**.
-7. In **Generate Rules**, select **Confirm**. The rules are generated.
-8. Select **Finish**. The [Agent Configuration][3] page displays the new rules.
-9. In [Agent Configuration][3], select **Deploy Agent Policy**.
+The default policy and its rules cannot be modified. You can use the policy priority setting to override the default policy with custom policies.
 
 
-## Create a custom Agent rule
+### Create a policy
```

Comment:
I found myself a bit confused about the sequence of actions on this page. The intro made it sound like creating a policy always happened first, but then it was hard to find this as a step 1 in the sidebar as a prerequisite for all the instructions for creating custom rules. Would it be possible to reinstate "policy" in an H2 somewhere in the page to make it easier to find?

Reply:
you create a policy and then either select or create rules to add to it. In many cases, the rules already exist. Like our immutable default rules.

But you're right that it isn't called out. Let me change it in a local build and then commit it.